examples/deployment-grpc-v2/02-contour.yaml

```diff
@@ -58,7 +58,15 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["wget", "-qO-", "--post-data=''", "http://localhost:9001/healthcheck/fail"] 
+              command:
+              - bash
+              - -c
+              - --
+              - echo
+              - -ne
+              - "POST /healthcheck/fail HTTP/1.1\r\nHost: localhost\r\nConnection:
```

Comment:
Don't know how this newline got in, but I think it will work without it. Weird.

Reply:
Done. PTAL